pkgs/native_doc_dartifier/test/dartify_simple_cases/dartified_snippets/identifiers.dart

```diff
@@ -0,0 +1,6 @@
+// AUTO GENERATED BY NATIVE_DOC_DARTIFIER. DO NOT EDIT!
```

Comment:
Document how to regenerate these files either directly in the comment here or if the instructions are too long and will be repetitive to paste them in each generated file, have them in some `CONTRIBUTION.md` file and point to it here.